app/lib/services.rb

```diff
@@ -30,4 +30,10 @@ def self.worldwide_api
   def self.registries
     Registries::BaseRegistries.new
   end
+
+  if ENV["FEATURE_FLAG_ACCOUNTS"] == "enabled"
+    def self.accounts_api
+      ENV.fetch("GOVUK_ACCOUNT_URI")
```

Comment:
What's the best naming convention here? endpoint? URI? URL?
Wasn't sure if this might end up in Plek and I should match what was there?